test/pytest/conftest.py

```diff
@@ -7,11 +7,13 @@
 
 import os
 import pytest
+from test_xpedite.test_pmu               import CPU_IDS
 from test_xpedite.test_profiler.scenario import ScenarioType
 
-PARAMETER_NAME = 'scenarioName'
+SCENARIO_PARAMETER_NAME = 'scenarioName'
```

Comment:
let's have a convention, Prefix all parameters with `PARAMATER`
rename `SCENARIO_PARAMETER_NAME` to `PARAMETER_SCENARIO`
rename `CPU_PARAMETER_NAME` to `PARAMETER_CPU_ID`